plugins/services/src/js/components/LogView.js

```diff
@@ -28,18 +28,12 @@ class LogView extends React.Component {
     });
 
     this.handleLogContainerScroll = Util.throttleScroll(
-      this.handleLogContainerScroll, 500
+      this.handleLogContainerScroll, 50
     );
 
     this.handleWindowResize = Util.debounce(
       this.handleWindowResize.bind(this), 100
     );
-
-    // Since we can receive a lot of updates (from a stream potentially),
-    // let's debounce this function to not call it too often
-    this.componentDidUpdate = Util.debounce(
-      this.componentDidUpdate.bind(this), 100
-    );
```

Comment:
I am working on a performance PR that will take care of this. More or less I am using `shouldComponentUpdate` a little more actively